task.sql

```diff
@@ -3,31 +1,55 @@
 CREATE DATABASE ShopDB;
 USE ShopDB;
 
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
-    PRIMARY KEY (ID)
+  ID INT,
+  Name VARCHAR(50),
+  
+  PRIMARY KEY (ID)
+);
+
+CREATE TABLE Warehouses (
+  ID INT,
```

Comment:
Consider adding `AUTO_INCREMENT` to the `ID` column to automatically generate unique IDs for new entries.

Reply:
Done